ui/app/AppLayouts/WalletV2/WalletHeader.qml

```diff
@@ -140,6 +229,479 @@ Item {
+                            recipientInput.valid = isValidEns || isValidAddress
+
+                            if (!recipientInput.valid || text == "") {
+                                sendTransactionModal.recipient = null
+                                validationError = qsTr("This is an invalid address")
+                            } else {
+                                sendTransactionModal.recipient = {
+                                    address: text
+                                }
+                                validationError = ""
+                            }
+                        }
+
+                        Rectangle {
+                            width: 36
+                            height: 24
+                            visible: !recipientInput.pasteFromClipboard
+                            anchors.bottom: parent.bottom
+                            anchors.bottomMargin: checkmarkIcon.visible ? 16 : 33
+                            anchors.right: parent.right
+                            anchors.rightMargin: 6
+                            color: Theme.palette.baseColor2
+                            StatusIcon {
+                                id: cancelIcon
+                                visible: !recipientInput.valid && !recipientInput.pasteFromClipboard
+                                icon: "cancel"
+                                color: Theme.palette.dangerColor1
+                                height: 16
+                                anchors.centerIn: parent
+                                anchors.rightMargin: 8
+                            }
+                            StatusIcon {
+                                id: checkmarkIcon
+                                visible: recipientInput.valid && !recipientInput.pasteFromClipboard
+                                icon: "tiny/checkmark"
+                                height: 16
+                                color: Theme.palette.primaryColor1
+                                anchors.centerIn: parent
+                                anchors.rightMargin: 8
+                            }
+                        }
+                    }
+
+                    Loader {
+                        id: selectedRecipientLoader
+                        anchors.top: parent.top
+                        anchors.topMargin: 18
+                        anchors.left: parent.left
+                        anchors.leftMargin: 20
+                        anchors.right: parent.right
+                        active: !!selectedRecipient
+                        property var selectedRecipient
+                        
+                        sourceComponent: StatusListItem {
+                            sensor.enabled: false
+                            anchors.left: parent.left
+                            anchors.right: parent.right
+                            statusListItemTitle.font.weight: Font.Medium
+                            icon.name: "filled-account"
```

Comment:
At the moment, this PR only support imported wallet accounts without custom emojis, so this icon type here is hardcoded.